core/controllers/admin.py

```diff
@@ -373,3 +394,21 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SslChallengeHandler(base.BaseHandler):
+    """Plaintext page for responding to SSL challenges."""
```

Comment:
Maybe you should call these "Plaintext page for responding to LetsEncrypt SSL challenges"? "SSL challenges doesn't seem like it's a general term